src/Output/Color.php

```diff
@@ -157,10 +136,6 @@ public static function style(string $name, array $style): void
             throw new InvalidArgumentException('Trying to set empty or invalid style');
         }
 
-        if (isset(static::$styles[$name]) || method_exists(static::class, $name)) {
-            throw new InvalidArgumentException('Trying to define existing style');
```

Comment:
in case of factory styles (eg info ok comment etc) if the user tries to override fg and bg to same value it makes text invisible/illegible so need to throw a exception here

Reply:
Done!